lib/manager/helmv3/artifacts.spec.ts

```diff
@@ -2,11 +2,12 @@ import { exec as _exec } from 'child_process';
 import _fs from 'fs-extra';
 import { join } from 'upath';
 import { envMock, mockExecAll } from '../../../test/exec-util';
-import { mocked } from '../../../test/util';
+import { loadFixture, mocked } from '../../../test/util';
 import { GlobalConfig } from '../../config/global';
 import type { RepoGlobalConfig } from '../../config/types';
 import * as docker from '../../util/exec/docker';
 import * as _env from '../../util/exec/env';
+import * as hostRules from '../../util/host-rules';
```

Comment:
We should refactor tests to use imports from `test/utils`.

And we should use the new fixtures for filesystem mocking.

Can be done in a future PR.